app/src/main/java/com/benefit/utilities/staticClasses/Converters.java

```diff
@@ -33,4 +33,15 @@ public static Boolean isSubstring(String s1,String s2){
         }
         return false;
     }
+
+    public static int getMetaCategoryFromCategoryId(long categoryId) {
```

Comment:
why do we need this method? the data is available in the category object

Reply:
Because here we hold a product object, and it only holds the category Id as an int, not the object. Is calling the category service with getCategoryById better?

Reply:
I think doing it via the categoryService makes more logical sense, because here you defined ids hard-coded, what if we create more categories which are not in this mapping (and we'll do that, I have assignment about it)? It means that for each category that we have, we need to update this mapping also, and it's a violation of the single source principle. So i think if you have the option to read the actual data from categoryService, it's better

Reply:
Done